<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: Poppins;
background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
	height: 40px;
	border-radius: 20px;
	border: 1px solid #00007f;
	background-color: white;
	color: #00007f;
	font-weight: 500;
	font-size: 18px;
}

QLabel {
		text-align: center;
	}</string>
   </property>
   <widget class="QPushButton" name="skipBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Omitir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Siguiente</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLbl">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="instLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12pt;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="loadingBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 1px solid #bbb;
    border-radius: 5px;
    background-color: #f3f3f3;
	border-radius: 10px;
}

QProgressBar::chunk {
    background-color: #00007f;
	border-radius: 10px;
}
</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="showVoltLbl">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16pt;
color: rgb(134, 134, 134)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>instLbl</zorder>
   <zorder>imgLbl</zorder>
   <zorder>loadingBar</zorder>
   <zorder>showVoltLbl</zorder>
   <zorder>skipBtn</zorder>
   <zorder>nextBtn</zorder>
   <zorder>backBtn</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
